packages/morpho-ts/src/time/time.ts

```diff
@@ -5,6 +5,10 @@ const toNumberish = (returnValue: bigint, initialValue: bigint | number) => {
 
 const UNITS = ["ms", "s", "min", "h", "d", "w", "mo", "y"] as const;
 export type TUnit = (typeof UNITS)[number];
+export type TPeriod = {
+  unit: TUnit;
+  duration: number;
+};
```

Comment:
imo we should either export all from `Time` and nowhere else or export all here but no type from `Time`

At first I would have said everything here but `Unit` is such a broad naming that I indeed prefer exporting only from `Time`